admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -146,17 +146,76 @@ pupilAccessArrangementsDataService.sqFindPupilsWithAccessArrangements = async (d
       type: TYPES.Int
     }
   ]
-  const sql =
-    `SELECT p.urlSlug, p.foreName, p.middleNames, p.lastName, p.dateOfBirth, aa.description
-  FROM ${sqlService.adminSchema}.pupilAccessArrangements paa
-    INNER JOIN ${sqlService.adminSchema}.pupil p
-      ON paa.pupil_id = p.id
-    INNER JOIN ${sqlService.adminSchema}.school s
-      ON p.school_id = s.id
-    INNER JOIN ${sqlService.adminSchema}.accessArrangements aa
-      ON aa.id = paa.accessArrangements_id
-  WHERE s.dfeNumber = @dfeNumber
-  ORDER BY p.lastName`
+  const sql = `
+    SELECT *, completedCheck & ~unusedRestart hasCompletedCheck
+    FROM (
+      SELECT p.urlSlug, p.foreName, p.middleNames, p.lastName, p.dateOfBirth, aa.description,
+      CASE WHEN unusedRestart.id IS NULL THEN 0 ELSE 1 END unusedRestart,
```

Comment:
Optionally you could cast these integers into bits so that they would appear as booleans in the JS:
```SQL
SELECT CAST(1 AS BIT); -- true
SELECT CAST(0 AS BIT); -- false 
```